staging/vhost-device-gpu/src/virt_gpu.rs

```diff
@@ -245,12 +362,12 @@ impl VirtioGpu {
         }
     }
 
-    pub fn force_ctx_0(&self) {
+    fn force_ctx_0(&self) {
         self.rutabaga.force_ctx_0()
     }
 
     /// Gets the list of supported display resolutions as a slice of `(width, height, enabled)` tuples.
```

Comment:
I think documentation comments should generally be part of the trait, not the specific implementation. I would suggest moving all the comments there.